src/quickfix.c

```diff
@@ -6514,6 +6549,9 @@ ex_vimgrep(exarg_T *eap)
     if (vgr_process_args(eap, &args) == FAIL)
 	goto theend;
 
+    if (eap->cmdidx == CMD_vimgrep && !check_can_set_curbuf_forceit(eap->forceit))
+	return;
```

Comment:
and here please

Reply:
We might be able to remove this one. Need to double-check

Reply:
okay thanks

Reply:
Done!